Dockerfile

```diff
@@ -2,7 +2,8 @@ FROM node:boron
 
 # Update and install packages
 RUN apt-get update && \
-    apt-get install -y python python-dev python-pip python-virtualenv zip rsync && rm -rf /var/lib/apt/lists/*
+    apt-get install -y python python-dev python-pip python-virtualenv zip rsync default-jdk && \
```

Comment:
@imujjwal96 try update and install https://stackoverflow.com/questions/34571711/cant-run-curl-command-inside-my-docker-container